[IMP] inventory: update cycle counts (#4464)
odoo/documentation: content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

Cycle counts
============

For most companies, warehouse stock only needs to be counted once a year. This is why, by default,

> after making an *inventory adjustment* in Odoo, the scheduled date for the next inventory count sets

Reviewer: the scheduled date for the next inventory count is set

for the 31st of December of the current year.

However, for some businesses, it's crucial to have an accurate inventory count at all times. These
companies use *cycle counts* to keep critical stock levels accurate. Cycle counting is a method by
which companies count their inventory more often in certain *locations*, to ensure that their

> physical inventory counts match their inventory records.

Reviewer: physical inventory counts match their inventory records in Odoo.

Not sure if necessary- but maybe just specifying even more that it is the data vs the physical.

> which companies count their inventory more often in certain *locations*, to ensure that their
> physical inventory counts match their inventory records in Odoo.

Reviewer 2: ```suggestion
which companies count their inventory more often in certain locations, to ensure that their
physical inventory counts match their inventory records in Odoo.
```
while I see the benefit of italicizing *inventory adjustments* and *cycle counts*, I'm not sure that same methodology should apply for locations.

Activate storage locations
==========================

> In Odoo, cycle counts are location-based. Therefore, the *storage locations* feature needs to be
> enabled before performing a cycle count.

Reviewer 2: ```suggestion
In Odoo, cycle counts are location-based. Therefore, the :guilabel:`Storage Locations` feature
needs to be enabled before performing a cycle count.
```
added a guilabel here because Storage Locations is the name of a feature.

To enable this feature, navigate to :menuselection:`Inventory app --> Configuration --> Settings`,
and scroll down to the :guilabel:`Warehouse` section. Then, click the checkbox next to
:guilabel:`Storage Locations`. Click :guilabel:`Save` to save all changes.

.. image:: cycle_counts/cycle-counts-enabled-setting.png
   :align: center
   :alt: Enabled storage locations setting in inventory settings.

> Change inventory frequency by location

Reviewer: Change inventory count frequency by location

Maybe specify it's the count here

============================================

> Now that the storage locations setting is enabled, the inventory count frequency can be changed for
> specific locations created in the warheouse.

Reviewer 2: ```suggestion
Now that the :guilabel:`Storage Locations` setting is enabled, the inventory count frequency can be
changed for specific locations created in the warheouse.
```

To view (and edit) locations, navigate to :menuselection:`Inventory app --> Configuration -->

> Locations`. This reveals a :menuselection:`Locations` page containing every location currently
> created and listed in the warehouse.

Reviewer 2: ```suggestion
Locations`. This reveals a :guilabel:`Locations` page containing every location currently created
and listed in the warehouse.
```
changing it to a guilabel here because the navigation to the page is already complete, and this is the title that appears on the page/screen/UI

From this page, click into a location to reveal the settings and configuration page for that
location. Click :guilabel:`Edit` to edit the location settings.

Under the :guilabel:`Cyclic Inventory` section, locate the :guilabel:`Inventory Frequency (Days)`
field, which should be set to `0` (if this location has not been edited previously). In this field,
change the value to whichever number of days is desired.

.. image:: cycle_counts/cycle-counts-location-frequency.png
   :align: center
   :alt: Location frequency setting on location.

.. example::
   A location that needs an inventory count every 30 days should have the :guilabel:`Inventory
   Frequency (Days)` value set to `30`.

Once the frequency has been changed to the desired number of days, click :guilabel:`Save` to save
changes. Now, once an inventory adjustment is applied to this location, the next scheduled count

> date will be automatically set based on the value entered into the :guilabel:`Inventory Frequency
> (Days)` field.

Reviewer 2: ```suggestion
date is automatically set, based on the value entered into the :guilabel:`Inventory Frequency
(Days)` field.
```
established present tense

Count inventory by location
===========================

To perform a cycle count for a specific location in the warehouse, navigate to
:menuselection:`Inventory app --> Operations --> Inventory Adjustments`. This reveals an
:guilabel:`Inventory Adjustments` page containing all products currently in stock, with each product
listed on its own line.

From this page, the :guilabel:`Filters` and :guilabel:`Group By` buttons (at the top of the page,

> under the :guilabel:`search bar`), can be used to select specific locations and perform inventory
> counts.

Reviewer 2: ```suggestion
under the :guilabel:`Search...` bar), can be used to select specific locations and perform
inventory counts.
```
i know that there's been some back and forth re: the search bar, and if it should have a guilabel. I think, if we're going to use one, we should be as accurate as possible and convey *exactly* what's seen on the screen - hence my suggestion...what do you think? 🤔

.. image:: cycle_counts/cycle-counts-inventory-adjustments-page.png
   :align: center
   :alt: Inventory adjustments page.

To select a specific location and view all products within that location, click :guilabel:`Group
By`, then click :guilabel:`Add Custom Group` to reveal a new drop-down menu to the right. Click this

> menu to reveal the drop-down, and select :guilabel:`Location`. Then, click :guilabel:`Apply`. The

Reviewer: Click :guilabel:`Location`  from the drop-down menu, then click :guilabel:`Apply`.

Reviewer: The drop-down was already revealed by clicking Add Custom Group, so I just wanted to clear that up and streamline this.

> To select a specific location and view all products within that location, click :guilabel:`Group
> By`, then click :guilabel:`Add Custom Group` to reveal a new drop-down menu to the right. Click
> :guilabel:`Location` from the drop-down menu, then click :guilabel:`Apply`. The page now displays
> condensed drop-down menus of each location in the warheouse that has products in stock, and a cycle
> count can be performed for all products in that location.

Reviewer 2: ```suggestion
To select a specific location, and view all products within that location, click :guilabel:`Group
By`, then click :guilabel:`Add Custom Group` to reveal a new drop-down menu to the right.

Click :guilabel:`Location` from the drop-down menu, then click :guilabel:`Apply`. The page now
displays condensed drop-down menus of each location in the warehouse that have products in stock,
and a cycle count can be performed for all products in that location.
```

.. tip::

>    In large warehouses with a large number of locations and high volume of products, it might be
>    easier to search for the specific location desired. To do this, from the
>    :menuselection:`Inventory Adjustments` page, click :guilabel:`Filters`. Then, click
>    :guilabel:`Add Custom Filter` to reveal a new menu to the right. Click this menu to reveal three
>    drop-downs.

Reviewer 2: ```suggestion
   In large warehouses, with a large number of locations and high volume of products, it might be
   easier to search for the specific location desired. To do this, from the :guilabel:`Inventory
   Adjustments` page, click :guilabel:`Filters`. Then, click :guilabel:`Add Custom Filter` to
   reveal a new menu to the right. This menu contains three drop-downs.
```
changed it to a guilabel for the same reason mentioned above

>    For the first field, click and select :guilabel:`Location` from the drop-down. For the second
>    field, leave the `contains` value as is. For the third field, type in the name of the location
>    that is being searched for. Click :guilabel:`Apply` for that location to appear on the page.

Reviewer 2: ```suggestion
   For the first field, click open the drop-down menu, and select :guilabel:`Location`. For the
   second field, leave the `contains` value as is. For the third field, type in the name of the
   location that is being searched for. Click :guilabel:`Apply` for that location to appear on the
   page.
```

>    that is being searched for. Click :guilabel:`Apply` for that location to appear on the page.

Reviewer: GREAT suggestion!

Author: Thank you!!

.. image:: cycle_counts/cycle-counts-filters.png
   :align: center
   :alt: Applied filters and group by on inventory adjustments page.

Change full inventory count frequency
=====================================

While cycle counts are typically performed per location, the scheduled date for full inventory
counts of everything in-stock in the warehouse can also be manually changed to push the date up

> sooner than is listed.

Reviewer 2: ```suggestion
sooner than the date listed.
```

By default, the scheduled date for inventory adjustments are always scheduled for the 31st of

> December of the current year. However, for some companies, it is crucial that they have an accurate
> inventory count at all times. In such cases, the default scheduled date can be modified.

Reviewer 2: ```suggestion
December of the current year. However, for some companies, it is crucial for them to have an
accurate inventory count at all times. In such cases, the default scheduled date can be modified.
```

To modify the default scheduled date, go to :menuselection:`Inventory app --> Configuration -->
Settings`. Then, in the :guilabel:`Operations` section, locate the :guilabel:`Annual Inventory Day
and Month` setting field, which includes a drop-down that is set to `31 December` by default.

.. image:: cycle_counts/cycle-counts-frequency-settings.png
   :align: center
   :alt: Frequency field in inventory app settings.

To change the day, click the :guilabel:`31`, and change it to a day within the range `1-31`,
depending on the desired month of the year.

Then, to change the month, click :guilabel:`December` to reveal the drop-down menu, and select the
desired month.

> Once all desired changes have been made, click :guilabel:`Save` to save all changes.

Reviewer 2: ```suggestion
Once all changes have been made, click :guilabel:`Save` to save all changes.
```

set for the 31st of December of the current year.

However, for some businesses, it's crucial to have an accurate inventory count at all times. These
companies use *cycle counts* to keep critical stock levels accurate. Cycle counting is a method by
which companies count their inventory more often in certain *locations*, to ensure that their
physical inventory counts match their inventory records in Odoo.

Activate storage locations
==========================

In Odoo, cycle counts are location-based. Therefore, the *Storage Locations* feature needs to be
enabled before performing a cycle count.

To enable this feature, navigate to :menuselection:`Inventory app --> Configuration --> Settings`,
and scroll down to the :guilabel:`Warehouse` section. Then, click the checkbox next to
:guilabel:`Storage Locations`. Click :guilabel:`Save` to save all changes.

.. image:: cycle_counts/cycle-counts-enabled-setting.png
   :align: center
   :alt: Enabled storage locations setting in inventory settings.

Change inventory count frequency by location
============================================

Now that the storage locations setting is enabled, the inventory count frequency can be changed for

> specific locations created in the warheouse.

Reviewer 3: Typo
```suggestion
specific locations created in the warehouse.
```

changes. Now, once an inventory adjustment is applied to this location, the next scheduled count
date is automatically set, based on the value entered into the :guilabel:`Inventory Frequency
(Days)` field.

Count inventory by location
===========================

To perform a cycle count for a specific location in the warehouse, navigate to
:menuselection:`Inventory app --> Operations --> Inventory Adjustments`. This reveals an
:guilabel:`Inventory Adjustments` page containing all products currently in stock, with each product
listed on its own line.

From this page, the :guilabel:`Filters` and :guilabel:`Group By` buttons (at the top of the page,
under the :guilabel:`Search...` bar), can be used to select specific locations and perform inventory
counts.

.. image:: cycle_counts/cycle-counts-inventory-adjustments-page.png
   :align: center
   :alt: Inventory adjustments page.

To select a specific location and view all products within that location, click :guilabel:`Group
By`, then click :guilabel:`Add Custom Group` to reveal a new drop-down menu to the right.

Click :guilabel:`Location` from the drop-down menu, then click :guilabel:`Apply`. The page now

> displays condensed drop-down menus of each location in the warheouse that has products in stock, and

Reviewer 3: Typo again
```suggestion
displays condensed drop-down menus of each location in the warehouse that has products in stock, and
```

a cycle count can be performed for all products in that location.

.. tip::
   In large warehouses with a large number of locations and high volume of products, it might be
   easier to search for the specific location desired. To do this, from the
   :menuselection:`Inventory Adjustments` page, click :guilabel:`Filters`. Then, click
   :guilabel:`Add Custom Filter` to reveal a new menu to the right. Click this menu to reveal three
   drop-downs.

   For the first field, click and select :guilabel:`Location` from the drop-down. For the second

>    field, leave the `contains` value as is. For the third field, type in the name of the location

Reviewer 3: I think this value should be :guilabel: since it is not user input
```suggestion
   field, leave the :guilabel:`contains` value as is. For the third field, type in the name of the
```

   that is being searched for. Click :guilabel:`Apply` for that location to appear on the page.

.. image:: cycle_counts/cycle-counts-filters.png
   :align: center
   :alt: Applied filters and group by on inventory adjustments page.

Change full inventory count frequency
=====================================

While cycle counts are typically performed per location, the scheduled date for full inventory
counts of everything in-stock in the warehouse can also be manually changed to push the date up
sooner than the date listed.

By default, the scheduled date for inventory adjustments is always scheduled for the 31st of
December of the current year. However, for some companies, it is crucial for them to have an
accurate inventory count at all times. In such cases, the default scheduled date can be modified.

To modify the default scheduled date, go to :menuselection:`Inventory app --> Configuration -->
Settings`. Then, in the :guilabel:`Operations` section, locate the :guilabel:`Annual Inventory Day

> and Month` setting field, which includes a drop-down that is set to `31 December` by default.

Reviewer 3: I think this value should be :guilabel: since it is not user input
```suggestion
and Month` setting field, which includes a drop-down that is set to :guilabel:`31 December` by default.
```

> .. image:: cycle_counts/cycle-counts-frequency-settings.png

Reviewer 3: This image filesize is pretty large, see if you can reduce it by retaking the image and compressing again. If you cant get it any smaller then you can leave it 🙂

Cycle counts
============

For most companies, warehouse stock only needs to be counted once a year. This is why, by default,
after making an *inventory adjustment* in Odoo, the scheduled date for the next inventory count is
set for the 31st of December of the current year.

However, for some businesses, it's crucial to have an accurate inventory count at all times. These
companies use *cycle counts* to keep critical stock levels accurate. Cycle counting is a method by
which companies count their inventory more often in certain *locations*, to ensure that their

> physical inventory counts match their inventory records in Odoo.

Reviewer 4: Defining a broader industry topic, not specific to Odoo
```suggestion
physical inventory counts match their inventory records.
```

(Days)` field.

Count inventory by location
===========================

To perform a cycle count for a specific location in the warehouse, navigate to
:menuselection:`Inventory app --> Operations --> Inventory Adjustments`. This reveals an
:guilabel:`Inventory Adjustments` page containing all products currently in stock, with each product
listed on its own line.

From this page, the :guilabel:`Filters` and :guilabel:`Group By` buttons (at the top of the page,
under the :guilabel:`Search...` bar), can be used to select specific locations and perform inventory
counts.

.. image:: cycle_counts/cycle-counts-inventory-adjustments-page.png
   :align: center
   :alt: Inventory adjustments page.

To select a specific location and view all products within that location, click :guilabel:`Group
By`, then click :guilabel:`Add Custom Group` to reveal a new drop-down menu to the right.

Click :guilabel:`Location` from the drop-down menu, then click :guilabel:`Apply`. The page now
displays condensed drop-down menus of each location in the warehouse that has products in stock, and
a cycle count can be performed for all products in that location.

.. tip::

>    In large warehouses with a large number of locations and high volume of products, it might be

Reviewer 4: Suggestion
```suggestion
   In large warehouses with a multiple locations and/or a high volume of products, it might be
```

   easier to search for the specific location desired. To do this, from the :guilabel:`Inventory
   Adjustments` page, click :guilabel:`Filters`. Then, click :guilabel:`Add Custom Filter` to reveal
   a new menu to the right. Click this menu to reveal three drop-downs.

   For the first field, click and select :guilabel:`Location` from the drop-down. For the second
   field, leave the :guilabel:`contains` value as is. For the third field, type in the name of the
   location that is being searched for. Click :guilabel:`Apply` for that location to appear on the
   page.

.. image:: cycle_counts/cycle-counts-filters.png
   :align: center
   :alt: Applied filters and group by on inventory adjustments page.

Change full inventory count frequency
=====================================

While cycle counts are typically performed per location, the scheduled date for full inventory
counts of everything in-stock in the warehouse can also be manually changed to push the date up
sooner than the date listed.

> By default, the scheduled date for inventory adjustments is always scheduled for the 31st of
> December of the current year. However, for some companies, it is crucial for them to have an
> accurate inventory count at all times. In such cases, the default scheduled date can be modified.

Reviewer 4: Redundant to the intro paragraph / not needed
```suggestion
```

   field, leave the :guilabel:`contains` value as is. For the third field, type in the name of the
   location that is being searched for. Click :guilabel:`Apply` for that location to appear on the
   page.

.. image:: cycle_counts/cycle-counts-filters.png
   :align: center
   :alt: Applied filters and group by on inventory adjustments page.

Change full inventory count frequency
=====================================

While cycle counts are typically performed per location, the scheduled date for full inventory
counts of everything in-stock in the warehouse can also be manually changed to push the date up
sooner than the date listed.

By default, the scheduled date for inventory adjustments is always scheduled for the 31st of
December of the current year. However, for some companies, it is crucial for them to have an
accurate inventory count at all times. In such cases, the default scheduled date can be modified.

To modify the default scheduled date, go to :menuselection:`Inventory app --> Configuration -->
Settings`. Then, in the :guilabel:`Operations` section, locate the :guilabel:`Annual Inventory Day

> and Month` setting field, which includes a drop-down that is set to :guilabel:`31 December` by

Reviewer 3: I know I previously suggested for this to all be guilabel, but I stand corrected since the day is user input while the month is a dropdown selection.
```suggestion
and Month` setting field, which includes a drop-down that is set to `31` :guilabel:`December` by
```

default.

.. image:: cycle_counts/cycle-counts-frequency-settings.png
   :align: center
   :alt: Frequency field in inventory app settings.

> To change the day, click the :guilabel:`31`, and change it to a day within the range `1-31`,
> depending on the desired month of the year.

Reviewer 3: Changing back user input was correct!
```suggestion
To change the day, click the `31`, and change it to a day within the range `1-31`, depending on the
desired month of the year.
```
